docs/_attic/WIP-lamborghini-distribution/transactions.md

```diff
@@ -57,7 +57,7 @@ func GetDelegatorEntitlement(delegatorAddr sdk.AccAddress) DecCoins
     
     return entitlement
 
-func (pc PowerChange) ProcessPowerChangeDelegation(delegation sdk.Delegation, 
+func (pc PowerChange) ProcessPowerChangeDelegation(delegation   Delegation, 
```

Comment:
I'd remove this file altogether really

Reply:
agree, I'd remove the `docs/_attic` completely

Reply:
@gamarin2 @jackzampolin can we remove it ?